dds/DCPS/transport/framework/TransportClient.h

```diff
@@ -35,7 +35,7 @@ namespace DCPS {
 
 class EntityImpl;
 class TransportInst;
-class AssocationInfo;
+class AssociationInfo;
```

Comment:
```suggestion
```
Evidently we don't need this?

Reply:
No and there's not even a class with that name anymore.

Reply:
Looking at all of the forward declarations in that group, it doesn't look like any of them are useful anymore. I wish the compilers or something else would help us with this.